### PR TITLE
Collection of smallish cleanups

### PR DESCRIPTION
**Pull request by jerith**
There are various PEP 8 violations and TODOs that can be cleaned up fairly easily. This is a ticket to collect those.

### CODE REVIEW

**Comment by jerith** [comment]
It's probably easiest to review this one commit at a time, although the only change that is spread across multiple files is 1182d4c.

File: go/vumitools/api.py [hunk 1]
```diff
@@ -181,9 +179,10 @@ def running_conversations(self):
 
     @Manager.calls_manager
     def draft_conversations(self):
-        # TODO: Get rid of this once the old UI finally goes away.
+        # TODO: This should probably be `stopped_conversations` instead, but we
+        #       still apparently use `draft` in the UI in places.
         conversations = yield self.active_conversations()
-        returnValue([c for c in conversations if c.is_draft()])
+        returnValue([c for c in conversations if c.stopped()])
```
Comment (justinvdm) [inline go/vumitools/api.py:185]: Doesn't seem like the right PR for this change, but if the tests pass maybe its ok.

Comment (justinvdm) [inline go/vumitools/api.py:185]: Ah, after reading the rest of the PR this makes a bit more sense.

**Comment by justinvdm** [comment]
:+1: once Travis is happy.

File: go/vumitools/api.py (hunk 1 above)
Comment (jerith) [inline go/vumitools/api.py:185]: `is_draft` doesn't have much meaning anymore and it's functionally equivalent to `stopped` if archived conversations are excluded, which is the case here.

**Pull request merged**